pages/home/index.html

```diff
@@ -17,6 +17,14 @@
   <script src="../../modules/leaflet/leaflet.js"></script>
   <!-- Global Styles -->
   <link rel="stylesheet" href="../../assets/css/_global.css">
+  <!-- Navbar CSS -->
+  <link rel="stylesheet" href="../../assets/css/navbar.css">
```

Comment:
the navbar css has to be imported inside _global.css, the rest of these styles can be imported in your feature styles ./style.css